Formula/ffmpeg.rb

```diff
@@ -95,6 +103,15 @@ def install
       --host-ldflags=#{ENV.ldflags}
     ]
 
+    if build.with? "openh264"
+      resource("openh264-1.5.0").stage do
+        system "make", "install-shared", "PREFIX=#{libexec}/openh264-1.5.0"
+        chmod 0444, "#{libexec}/openh264-1.5.0/lib/libopenh264.dylib"
```

Comment:
Very unlike you to use `#{libexec}` instead of `libexec` ILZ, are you feeling okay? :trollface: